contracts/mainnet/connectors/notional/helpers.sol

```diff
@@ -0,0 +1,272 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
```

Comment:
use this in here https://github.com/Instadapp/dsa-connectors/blob/main/contracts/mainnet/common/math.sol

Reply:
done